pkg/ansibleapp/bind.go

```diff
@@ -0,0 +1,131 @@
+package ansibleapp
+
+import (
+	b64 "encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+)
+
+// TODO: Figure out the right way to allow ansibleapp to log
+// It's passed in here, but that's a hard coupling point to
+// github.com/op/go-logging, which is used all over the broker
+// Maybe ansibleapp defines its own interface and accepts that optionally
+// Little looser, but still not great
+func Bind(
+	instance *ServiceInstance,
+	parameters *Parameters,
+	clusterConfig ClusterConfig, log *logging.Logger,
+) (*BindData, error) {
+	log.Notice("============================================================")
+	log.Notice("                       BINDING                              ")
+	log.Notice("============================================================")
+	log.Notice(fmt.Sprintf("Parameters: %v", parameters))
+	log.Notice("============================================================")
+
+	var client *Client
+	var err error
+
+	if client, err = NewClient(log); err != nil {
+		return nil, err
+	}
+
+	if err = client.PullImage(instance.Spec.Name); err != nil {
+		return nil, err
+	}
+
+	output, err := client.RunImage("bind", clusterConfig, instance.Spec, parameters)
+
+	if err != nil {
+		log.Error("Problem running image", err)
+		return nil, err
+	}
+
+	log.Info("{" + string(output) + "}")
+
+	log.Notice("Calling getPodName")
```

Comment:
I'd change these to Debug level but keep them in so they can be turned on and off by us.